examples/optimize_evoked.py

```diff
@@ -0,0 +1,81 @@
+"""
+=================================================
+08. Optimize simulated evoked response parameters
+=================================================
+
+This example demonstrates how to optimize the parameters
+of the model simulation to match an experimental dipole waveform.
+"""
+
+# Authors: Blake Caldwell <blake_caldwell@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+import os.path as op
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Let us import hnn_core
+
+import hnn_core
+from hnn_core.dipole import simulate_dipole
+from hnn_core import read_params, Dipole, MPIBackend, Network
```

Comment:
```suggestion
from hnn_core import read_params, Dipole, MPIBackend, jones_2009_model
```

Reply:
ugh ... didn't seem to have fixed it :(

Reply:
Note to self: I wonder if there is some mismatch in the units of dipole that is read in vs the simulate dipole